packages/contracts/package.json

```diff
@@ -26,9 +26,11 @@
     "test:contracts": "cross-env SOLPP_FLAGS=\"FLAG_IS_TEST,FLAG_IS_DEBUG\" buidler test --show-stack-traces",
     "coverage": "yarn run coverage:contracts",
     "coverage:contracts": "cross-env SOLPP_FLAGS=\"FLAG_IS_TEST\" buidler coverage --network coverage --show-stack-traces --testfiles \"test/contracts/**/*.spec.ts\"",
-    "build": "yarn run build:contracts && yarn run build:typescript",
+    "build": "yarn run build:contracts && yarn run build:typescript && yarn run build:copy",
     "build:contracts": "buidler compile",
     "build:typescript": "tsc -p .",
+    "build:copy": "yarn run build:copy:contracts",
+    "build:copy:contracts": "copyfiles -u 2 \"contracts/optimistic-ethereum/**/*.sol\" \"build/contracts\"",
```

Comment:
What is this needed for?

Reply:
It's useful for being able to import the contracts into other packages